### ISSUE TITLE
Allow Ammonite to be managed by SDKMAN!

### ISSUE THREAD

**Issue by hammer** [lihaoyi/Ammonite#1029]
I have found https://sdkman.io to be a very useful tool to switch between the various Java, Scala, and sbt versions required by different projects we work on. I consider Ammonite part of the Scala platform for our developers, and I would love for SDKMAN! to be able to manage our Ammonite installations as well.

It appears that this possibility was discussed in the past (cf. https://github.com/sdkman/sdkman-db-migrations/issues/33). However, I was not able to find an archive of this discussion.

Any chance we could re-open the discussion for allowing SDKMAN! to manage Ammonite, or at least document why it's not a good idea in a public forum?

**Comment by lihaoyi** [comment]
no idea what sdkman is, feel free to add support, but I don't think that's something that Ammonite needs to care about. Ammonite packages for homebrew, apt-get, chocolatey, etc. all live in their respective repos

**Comment by ganeshchand** [comment]
brew install ammonite works great. However, where it gets tricky is that it will pick the version for us and I ran into an issue recently where my ammonite was broken because the libraries I specified in predef.sc file are not yet available for Scala 2.13.
I had to then go to ammonite website, find the ammonite install URL for 2.12 and install it.

sdkman is a great tool for JVM ecosystem, similar to . It let's us pick the specific version, can show all available versions and allow us to switch between the version. I can switch from Scala 2.13 to 2.12 on my computer with just one command. It would be great to see ammonite also available in sdkman.

I got the exact uplifting experience going from brew to sdkman as I did going from pip to conda. :-)

**Comment by ganeshchand** [comment]
Also, for newbies, who are just trying to get started with scala, the setup instructions would be as simple as running following 4 commands in the terminal. No more downloading zip file, unzipping, setting up path, which is where beginners tend to struggle.

```bash
curl -s "https://get.sdkman.io" | bash # install sdk manager
sdk install java 11.0.11.j9-adpt
sdk install scala 2.13.6
# sdk install ammonite ??
```
